composer.json

```diff
@@ -11,7 +11,7 @@
 		"ext-sodium": "*",
 
 		"silex/silex": "~2.0",
-		"twig/twig": "~1.23",
+		"twig/twig": "~2.0",
```

Comment:
I bet I missed/forgot about the discussion about this in the daily or so, but why is this 2.0 and not 3.0 here?

Reply:
Oops, in my review I checked all the namespace and Object updates and missed this. Good catch!

Reply:
Brilliant catch, thanks!

I guess I changed it at some point to be compatible with fundraising-content-provider and forgot to bump it.

Twig 3.0 brings some internal changes, so I'm setting this to [DNM]